release-team/role-handbooks/docs/README.md

```diff
@@ -1,387 +1,543 @@
+1.  Read the release timeline and **make sure the timeline includes deadlines for documentation work**, e.g: [1.14 timeline](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.14#timeline): 
+ 
+     - Docs deadline - Open placeholder PRs (~3.5 weeks before release)
+     - Docs deadline - PRs ready for review: (~2 weeks before release)
+     - Docs complete - All PRs reviewed and ready to merge (~1 week before release)
 
-8. Add contacts to the shadows release docs. Example: [https://bit.ly/k8s114-contacts](https://docs.google.com/spreadsheets/d/1BiGSLuCqjglQS1bJvpKk6rKFMciebPkUndzgDRnJsns/edit?ts=5c3bd42a#gid=0)
+     If these deadlines aren't listed in the release timeline, request that the Release Lead add them. 
 
-9. You need push access to the k/website repo (contact the sig-docs chair if you don't have it)
+1. Introduce yourself to the current localization owners to sync up early on strategy (needs coordination for main release). e.g: https://github.com/kubernetes/website/issues/12396.
 
-10. IMPORTANT: Create a working branch (Named `dev-[future release]`, for example `dev-1.14`) locally, based on master. Push it to upstream.
+1. ⚠️ Attend the release team meeting outlined in the current release. If you cannot attend, a Shadow needs to attend in your place.
 
-- Open a pull request against `master` with a new branch that follows the naming convention `dev-X.Y`.
+1. Read this to learn more about the entire release team and process: https://github.com/kubernetes/sig-release/tree/master/release-team
 
-    For example, the [Release 1.14](https://github.com/kubernetes/website/pull/12584) PR uses the branch `dev-1.14`.
+1. Select your Shadows
 
-    This release pull request (also known as the [integration branch]) serves as the base for individual, component enhancement PRs of the release. A [integration branch] lets you bundle and merge multiple PRs simultaneously.
+    You will be provided with survey results from people interested working on the release team. If you do not have the results, contact the release lead for more informaiton.
 
-- Add the label `do-not-merge/hold` to the PR.
+    After vetting the volunteers for their roles, role leads should make a final decision on selected shadows with the incoming Release Team Lead. **In the past the SIG Docs release team has been between 3-6 members.**
+    
+    ⚠️ Beyond meeting the basic requirements / commitments, a good Shadow is someone who is active in sig-docs. Seleciton priority should go to previous shadows who want to continue their journey (to one day eventually lead). Other than those few guidelines, use your best judgement!
+     
```

Comment:
Beyond meeting the basic requirements and time commitments, ... SIG-Docs
spelling: Selection
possible edit:  Selection priority should go to previous shadows who want to eventually lead a SIG-Docs release.